src/main/java/com/stack/project/App.java

```diff
@@ -8,6 +8,5 @@ public class App
 {
     public static void main( String[] args )
     {
```

Comment:
please write a comment inside the main function to explain why it is empty.

Reply:
fixed